modules/geolocation/outputanalysis.py

```diff
@@ -0,0 +1,40 @@
+import numpy as np
+
+def get_best_location(array):
+
+    tempArrayOne=array[:,[2]].copy()
```

Comment:
Formatting nitpick: put spaces between the assignment operator (`=`) e.g. `b = 5`.